bugwarrior/services/github.py

```diff
@@ -278,29 +293,34 @@ def get_involved_issues(self, user):
         """ Grab all 'interesting' issues """
         issues = {}
         for issue in self.client.get_involved_issues(user):
-            url = issue['html_url']
-            tag = re.match('.*github\\.com/(.*)/(issues|pull)/[^/]*$', url)
-            if tag is None:
-                log.critical(" Unrecognized issue URL: %s.", url)
-                continue
-            issues[url] = (tag.group(1), issue)
+            try:
+                url = issue['html_url']
+                repo = self.get_repository_from_issue(issue)
+                issues[url] = (repo, issue)
+            except ValueError as e:
+                log.critical(e)
```

Comment:
Ah. I think moving the surrounding lines out of the try/except block would make that clear. Also I consider it best practice to have the bear minimum in the `try` block to avoid catching exceptions you didn't intend to, especially when using built-in exceptions. I'd suggest:

```py
url = issue['html_url']
try:
    repo = self.get_repository_from_issue(issue)
except ValueError as e:
    log.critical(e)
else:
    issues[url] = (repo, issue)
```